### PR TITLE
Feature/remove jquery

### LINKED ISSUE #5: Remove jQuery as a dependency
jQuery is only needed to load the mountpoint for aletheia. As we support >ie11, we can remove it to make the build less heavy and better integratable. Please use the DOM Query Selector for that

### PR DESCRIPTION
**Pull request by AndyOGo**
fixes #5 

- removed ~~jquery~~
- switch to selection based on `.class` not `[data-attributes]`
- utilized new dataset api

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -23,6 +23,7 @@
 </script>
   <div id="root"
        style="height: 100%"
+       class="o-aletheia-mount"
```
Comment (LucaMele) [inline src/index.html]: would it be not better to introduce here as well `js-` as prefix?

Comment (LucaMele) [reply src/index.html]: Sepratation of concern

Comment (AndyOGo) [reply src/index.html]: sure, I completely forgot...

**Comment by AndyOGo** [comment]
@LucaMele 
@lina-salih 
I just recognised that this plugin is inside `src/jquery/aletheia-plugin.js`, but since I just removed jQuery this does not makes sense any more. I would rather move this file directly to `src/`

**Pull request merged**